packaging/installer/kickstart-ng.sh

```diff
@@ -0,0 +1,843 @@
+# Static build install code
+
+set_static_archive_urls() {
+  if [ "${RELEASE_CHANNEL}" = "stable" ]; then
+    latest="$(download "https://api.github.com/repos/netdata/netdata/releases/latest" /dev/stdout | grep tag_name | cut -d'"' -f4)"
+    export NETDATA_STATIC_ARCHIVE_URL="https://github.com/netdata/netdata/releases/download/${latest}/netdata-${SYSARCH}-${latest}.gz.run"
+    export NETDATA_STATIC_ARCHIVE_CHECKSUM_URL="https://github.com/netdata/netdata/releases/download/${latest}/sha256sums.txt"
+  else
+    export NETDATA_STATIC_ARCHIVE_URL="${NETDATA_TARBALL_BASEURL}/netdata-latest.gz.run"
+    export NETDATA_STATIC_ARCHIVE_CHECKSUM_URL="${NETDATA_TARBALL_BASEURL}/sha256sums.txt"
+  fi
+}
+
+try_static_install() {
+  set_static_archive_urls "${RELEASE_CHANNEL}"
+  progress "Downloading static netdata binary: ${NETDATA_STATIC_ARCHIVE_URL}"
+
+  if ! download "${NETDATA_STATIC_ARCHIVE_URL}" "${tmpdir}/netdata-${SYSARCH}-latest.gz.run"; then
+    warning "Unable to download static build archive for ${SYSARCH}."
+    return 2
+  fi
+
+  if ! download "${NETDATA_STATIC_ARCHIVE_CHECKSUM_URL}" "${tmpdir}/sha256sum.txt"; then
+    fatal "Unable to fetch checksums to verify static build archive."
+  fi
+
+  if ! grep "netdata-${SYSARCH}-latest.gz.run" "${tmpdir}/sha256sum.txt" | safe_sha256sum -c - > /dev/null 2>&1; then
+    fatal "Static binary checksum validation failed. Stopping Netdata Agent installation and leaving binary in ${tmpdir}. Usually this is a result of an older copy of the file being cached somewhere upstream and can be resolved by retrying in an hour."
+  fi
+
+  if [ "${INTERACTIVE}" -eq 0 ]; then
+    opts="${opts} --accept"
+  fi
+
+  progress "Installing netdata"
+  # shellcheck disable=SC2086
+  if ! run ${ROOTCMD} sh "${tmpdir}/netdata-${SYSARCH}-latest.gz.run" ${opts} -- ${NETDATA_AUTO_UPDATES:+--auto-update} ${NETDATA_INSTALLER_OPTIONS}; then
+    fatal "Failed to install static build of Netdataon ${SYSARCH}."
+  fi
+
+  install_type_file="/opt/netdata/etc/netdata/.install-type"
+  if [ -f "${install_type_file}" ]; then
+    # shellcheck disable=SC1090
+    . "${install_type_file}"
+    cat > "${install_type_file}" <<- EOF
+	INSTALL_TYPE='kickstart-static'
+	PREBUILT_ARCH='${PREBUILT_ARCH}'
+	EOF
+  fi
+}
+
+# ======================================================================
+# Main program
+
+setup_terminal || echo > /dev/null
+
+while [ -n "${1}" ]; do
+  case "${1}" in
+    "--help")
+      usage
+      exit 0
+      ;;
+    "--no-cleanup") NO_CLEANUP=1 ;;
+    "--dont-wait") INTERACTIVE=0 ;;
+    "--non-interactive") INTERACTIVE=0 ;;
+    "--interactive") INTERACTIVE=1 ;;
+    "--stable-channel") RELEASE_CHANNEL="stable" ;;
+    "--no-updates") NETDATA_AUTO_UPDATES="" ;;
+    "--auto-update") NETDATA_AUTO_UPDATES="1" ;;
+    "--disable-telemetry") NETDATA_DISABLE_TELEMETRY="0" ;;
+    "--reinstall") NETDATA_REINSTALL=1 ;;
+    "--claim-only") NETDATA_NO_UPDATE=1 ;;
```

Comment:
It's difficult to associate the name of the variable with the option. Can we rename it to `CLAIM_ONLY` or something similar?

Reply:
@Ferroin Some notes have been marked as resolved, I'm "unmarking" them so that I can track the active/inactive parts the review.

Reply:
Fixed.